### PR TITLE
refactor: sample structure.

### PR DESCRIPTION
**Pull request by Lavanyagaur22**
After this PR gets merged, will release a new version of the library (v0.4.0).

### CODE REVIEW

File: offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt [hunk 1]
```diff
@@ -91,12 +91,10 @@ class ConflictInterceptor(private val conflictResolutionImpl: ConflictResolution
      */
     inner class OffixConflictCallback(
```
Comment (harshithdwivedi) [inline offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt]: Make this a different class if possible

File: offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt [hunk 2]
```diff
@@ -141,17 +139,15 @@ class ConflictInterceptor(private val conflictResolutionImpl: ConflictResolution
      */
     inner class OfflineCallback(
```
Comment (harshithdwivedi) [inline offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt]: Same here

**Review by harshithdwivedi — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by Lavanyagaur22** [comment]
@harshithdwivedi I checked out, codacy errors are nothing to worry about. So you can review and merge it :)

File: offix/src/main/java/org/aerogear/offix/callbacks/OffixConflictCallback.kt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import android.util.Log
+import com.apollographql.apollo.api.Error
+import com.apollographql.apollo.exception.ApolloException
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import org.aerogear.offix.ConflictResolutionHandler
+import org.aerogear.offix.conflictedMutationClass
+import org.aerogear.offix.interfaces.ConflictResolutionInterface
+
+/*Callback class which handles conflicts.
+ */
```
Comment (wtrocki) [inline offix/src/main/java/org/aerogear/offix/callbacks/OffixConflictCallback.kt:12]: I like that callback is file on it is own. this makes easy to see/replicate and link from docs.

**Review by harshithdwivedi — APPROVED** [review]

**Pull request merged**